Assets/Scripts/Fighting/SpellCasting.cs

```diff
@@ -13,6 +13,12 @@ public class SpellCasting : MonoBehaviour
         public int spellIndex;
         private Spell selectedSpell;
         private GameObject selectedSpellPrefab;
+        
+        public string keyUp = "player1_shoot_up";
+        public string keyDown = "player1_shoot_down";
```

Comment:
I feel like, there should actually be an enum for the different known keybindings, instead of string declarations
in classes, where it is needed. Seems nicer and more maintainable.